teachopencadd/talktorials/T018_automated_cadd_pipeline/talktorial.ipynb

```diff
@@ -76,7 +76,7 @@
     "- [Binding site detection](#Binding-site-detection) \n",
     "- [Chemical similarity](#Chemical-similarity)\n",
     "- [Molecular docking](#Molecular-docking)\n",
-    "- [Protein-ligand interactions](#Protein–ligand-interactions)\n",
+    "- [Protein&mdash;ligand interactions](#Protein&mdash;ligand-interactions)\n",
```

Comment:
@Armin-Ariamajd have you checked if this renders correctly in HTML on the website (`make html` in the `docs` folder)? If yes, great, I do prefer the mdash.

Reply:
I'm not exactly sure how to do that; could you please explain a bit more?